src/js/components/TaskFileListComponent.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react/addons";
+import MesosActions from "../actions/MesosActions";
+import MesosEvents from "../events/MesosEvents";
+import MesosStore from "../stores/MesosStore";
+
+var TaskFileListComponent = React.createClass({
+  displayName: "TaskFileListComponent",
+  propTypes: {
+    task: React.PropTypes.object.isRequired
+  },
+
+  getInitialState() {
+    var task = this.props.task;
+    var taskId = task.id || task.taskId;
+    return {
+      files: MesosStore.getTaskFiles(taskId)
+    };
+  },
+
+  componentWillMount() {
+    MesosStore.on(MesosEvents.CHANGE, this.onMesosChange);
+  },
+
+  componentDidMount() {
+    var task = this.props.task;
+    var agentId = task.slaveId;
+    var taskId = task.id || task.taskId;
+    if (this.state.files == null) {
+      MesosActions.requestTaskFiles(agentId, taskId);
+    }
+  },
+
+  componentWillUnmount() {
+    MesosStore.removeListener(MesosEvents.CHANGE, this.onMesosChange);
+  },
+
+  onMesosChange() {
+    var task = this.props.task;
+    var taskId = task.id || task.taskId;
+    var files = MesosStore.getTaskFiles(taskId);
+    if (files != null) {
+      this.setState({
+        files: files
+      });
+    }
+  },
+
+  getFileNodes() {
+    var files = this.state.files;
+    if (files != null) {
+      return files.map((file) => {
```

Comment:
2 small things :
- for single arguments, you can skip the brackets
- for direct return statements, you can skip the curlies

So this is preferrable:
```
return files.map(file => (
  <tr key={file.path}>
// ...
));
```